guides/leon/vsftpd.md

```diff
@@ -0,0 +1,211 @@
+# VSFTP GUIDE

```

Comment:
You may not see it yet but there is a thing called front matter which jekyll uses and it will need to be in your guide for the site to work. Use

---
layout: post
title: CentOS E-Commerce
author: Varghese Emlin Charly
---